### DISCUSSION TITLE
Happy New Year!

### DISCUSSION THREAD

**Discussion by Alexey174** [xyflow/xyflow#1804] [General]
Moritz, I wish you a happy New Year! Thank you so much for your development!
I wish you to achieve great success and earn a lot of money!

[Screenshot](https://user-images.githubusercontent.com/18084300/147799844-8009dba5-8055-4ae1-b593-cb8be9b290ea.png)

**Comment by moklick** [comment]
Hey @Alexey174 thanks for you words 🙏  Your projects looks awesome! What is it about?

**Reply by Alexey174** [reply]
Hi. This is a constructor for automating the business processes of marketers in social networks or messengers
